src/content/ja/ai-capabilities/prompt-generation.md

```
@@ -5,9 +5,71 @@ section: ai-capabilities
+
+とはいえ、毎度これを手書きするのは面倒なので、ChatGPTなどのLLMを使います。「この日本語プロンプトをFLUX.2用に詳しくして」「構図・ライティング・カメラ情報を足して整形して」「このプロンプトをQwen-Image用に整形して」といった雑な依頼でも、プロンプトの密度を押し上げるには十分です。
+
+画像生成モデルによっては専用のLLMを用意している場合もありますが、そこまで大きく改善されるわけではありません。あくまで画像生成モデルの性能のほうが重要です。
+
+---
+
+## ComfyUIでの運用
+
+ComfyUIでローカルに動かせるLLMもいくつかありますが、**APIノードでGeminiやChatGPTを呼ぶ**ことも検討してみてください。
+
+![](https://gyazo.com/94496f33d758475aa62f614978ea2252){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)